### ISSUE TITLE
A small number of REF_TYPE='T2' sources have a fake duplicate

### ISSUE THREAD

**Issue by geordie666** [legacysurvey/legacysurvey#156]
Tom Dwelly noticed some (88) duplicated Tycho/Gaia objects in DR10. We should document this as a possible issue in some of the earlier releases (likely DR7, DR8, DR9 should be sufficient).

An example from Dustin:

https://www.legacysurvey.org/viewer-dev/?ra=82.1018&dec=-57.5072&layer=ls-dr10a&zoom=15&sources-dr10a&tycho2&gaia-edr3&mark=82.10387845,-57.50575588;82.10460731,-57.50678378

Information from Dustin:

This busy view shows the two entries marked in yellow, the one Gaia-EDR3 entry in blue-green, and you can see that there are two Tycho-2 stars labelled.

The Tycho-2 entries are 8526-175-{1,2}.  According to the Tycho-2 paper,

"Each Tycho-2 entry, e.g. TYC 5311–1245–1, has a set of
three identification numbers: TYC1, TYC2 and TYC3. TYC1
is the GSC region number, TYC2 is the running number within
the region, and TYC3 is a Tycho specific component number.
Double star components will either have two different running
numbers or the same running number and different component
numbers."

so it sounds like Tycho-2 thinks it's a double star system.  From the images, that doesn't seem to be the case!  And the mags of those two entries are very similar:
    http://simbad.cds.unistra.fr/simbad/sim-id?Ident=TYC++8526+175+1&NbIdent=1&Radius=2&Radius.unit=arcmin&submit=submit+id
    http://simbad.cds.unistra.fr/simbad/sim-id?Ident=TYC++8526+175+2&NbIdent=1&Radius=2&Radius.unit=arcmin&submit=submit+id

the Gaia proper motion for that star is ~only in the Dec direction, so I don't think it's a case where large motion has split it into two sources -- I think it's just an error in the Tycho-2 catalog!

I was not aware of such issues in Tycho-2!

**Comment by weaverba137** [comment]
I believe this is mentioned as a known issue for DR10, @geordie666, can you confirm that we can close this once PR #165 is merged?

**Comment by geordie666** [comment]
This is a different issue, I think. It's an issue with Tycho itself, rather than the new issue we've documented for DR10. Really, this isn't a "known issue" with the Legacy Surveys at all — it's a "feature" of the Tycho catalogs. I'll think carefully about where to add this in the documentation, but it isn't a DR10-specific issue and shouldn't (necessarily) block DR10.

**Comment by moustakas** [comment]
I agree this isn't a DR10 blocking factor but we can and should fix this for DR11. For example, in this script we flag Tycho-2 stars which are actually galaxies--
  `/global/cfs/cdirs/cosmo/staging/tycho2/tycho2-galaxies.py`

We could regenerate the input Tycho 2 catalog we use via this script to remove duplicates. I wonder if there are any others...

**Comment by weaverba137** [comment]
Not sure whether to close this one after #165.

**Comment by geordie666** [comment]
Let's leave this open for now. Hopefully we can have a discussion on a future telecon about whether this is really an issue with the Legacy Surveys or just a feature of Tycho. I'm leaning towards including this in "updates" once we correct for the effect (i.e. perhaps for DR11) rather than documenting it as an issue.

**Comment by weaverba137** [comment]
That's fine, thank you.

**Comment by geordie666** [comment]
This was mostly addressed in PR #196. I'll leave this ticket open, though, in case we want to detail more information on what (if anything) we did for DR11 to address duplicated Tycho sources.
